Cloud account - User invite feature added (#540)
OctopusDeploy/docs: docs/administration/managing-users-and-teams/index.md

:::

## Inviting Users {#Managingusersandteams-Invitingusers}
:::hint

> Do you have an OctoID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud) to invite a user.

Reviewer: resolving validation issue.
```suggestion
Do you have an OctoID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.
```

> Do you have an OctoID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.

Reviewer 2: ```suggestion
Do you have an Octopus ID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.
```

Reviewer: Committing this suggestion so I can tweak it from a question to a directive :)

> Do you have an Octopus ID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.

Reviewer: ```suggestion
If you have an Octopus ID and a Cloud instance, you can follow these instructions to invite users to your [Octopus Cloud instance](/docs/octopus-cloud/index.md).
```
